app/models/contact_form.rb

```diff
@@ -8,6 +8,8 @@ class ContactForm < MailForm::Base
   attribute :to_email, validate: :validate_to_email
   attribute :subject, validate: true
 
+  attr_accessor :logged_in_email
```

Comment:
Where is this value being written?

Reply:
It is written from contacts_controller. It was done earlier for WebsiteContact, but I missed to add it to ContactWct. I've added it now.